src/modules/address-book/ModuleAddressBook.vue

```diff
@@ -119,8 +119,14 @@ export default {
               nickname: 'My Address',
               resolverAddr: ''
             }
-          ]
-        : [].concat(this.addressBookStore);
+          ].concat(this.addressBookStore)
+        : [
+            {
+              address: toChecksumAddress(this.$store.state.wallet.address),
```

Comment:
can you import the address from the state instead? change it for all the files as well.